rep-0003.rst

```diff
@@ -343,6 +343,54 @@ Build System Support:
 
 - Same as Indigo
 
+Noetic Ninjemys (May 2020 - May 2025)
+--------------------------------------
+Required Support for:
+
+- Ubuntu Focal Fossa (20.04)
+
+Recommended Support for:
+
+- Debian Buster
+- Fedora 32
+
+Architectures Supported:
+
+- amd64
+- arm32
+- arm64
+
+Targeted Languages:
+
+- C++14
+- Python 3.8
+  - Testing against Python 3.7 is recommended, but not required.
```

Comment:
Since Debian Buster ships with Python 3.7 any Python code needs to work with 3.7 as well as 3.8.

Reply:
ceb35d0 clarifies that Python 3.7 testing is necessary for Buster only.